package.json

```diff
@@ -33,10 +37,12 @@
     "lint": "yarn lint:eslint && yarn lint:prettier",
     "lint:eslint": "eslint --config .eslintrc.js .",
     "lint:prettier": "prettier --check \"{src,test,scripts}/**/**.ts\"",
+    "lint:plugin": "expo-module lint plugin",
```

Comment:
I think this might not be needed. We can add the plugin to `{projectRoot}tsconfig.json`

```
    "plugin/**/*.ts",
```

and then if needed any exception or lint changes can go to `.eslintrc.js`.